Rear Rider/Rear Rider/ConfigurationOptions/OptionsView.swift

```diff
@@ -21,56 +21,79 @@ extension Binding {
 }
 
 struct OptionsView: View {
-    @ObservedObject var conf = UserConfig()
+    @EnvironmentObject var conf: UserConfig
     @State var confAudio: String = ""
-    @State var confLightPattern: String = ""
+    @State var confLightPattern: Int = 1
     @State var confLightBrightness: Int = 1
     @State var confLightColor: Color = .white
     
+    @State var configChanged = false
```

Comment:
Is there any reason to have this boolean? It doesn't look like it's actually doing anything. Why are not not saving the config when the user changes it anymore?

Reply:
This variable changes to true when the user makes a change in the settings and activates the Save button. Previously you were calling the save function each time a change was made. This way when you are satisfied with the changes you can click the Save button and the settings will be saved and sent to the Pi.

Reply:
Yeah, I noticed that in a comment further down. See me thoughts on it there.